test/test_fx_experimental.py

```diff
@@ -1547,25 +1548,25 @@ def test_type_matches(self):
             (Optional[list[int]], list[int]),
         ] + [
             # pre-PEP585 signatures
-            (list[int], int),
-            (list[int], create_type_hint([int, int])),
-            (list[int], create_type_hint((int, int))),
-            (list[torch.Tensor], create_type_hint([torch.Tensor, torch.Tensor])),
+            (typing.List[int], int),
```

Comment:
I'd prefer you import List, saves a few characters for future readers :p

Reply:
To keep RUFF from complaining this it will need a `noqa: UP006` (coming in another PR). If we import it then we'll need the noqa in two places (on the import AND here).